dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -918,7 +923,7 @@ def __init__(self, *args, **kwargs):
     -------
     ``DALIDataset`` object based on DALI pipeline and compatible with ``tf.data.Dataset`` API.
 
-    """
+    """ # noqa E501
```

Comment:
I'd prefer, that the lines that are too long be broken, instead of turning off the check. Is this possible?